backend/users/migrations/0022_user_address_upper_index.py

```
@@ -0,0 +1,86 @@
+and migration state agree on every backend and `makemigrations --check` stays
+clean.
+"""
+
+import django.db.models.functions.text
+from django.db import migrations, models
+
+
+INDEX_NAME = 'users_user_address_upper_idx'
+
+
+def _index_is_invalid(schema_editor):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_index_is_invalid`

(ANN202)